packages/sync-core/api-report.md

```diff
@@ -201,6 +201,18 @@ export interface RoomSnapshot {
     tombstones?: Record<string, number>;
 }
 
+// @public (undocumented)
+export interface RoomStoreMethods {
```

Comment:
Could we make this `RoomStoreMethods<R>` instead of always returning/accepting `UnknownRecord`?